pkg/util/config/feature_gate.go

```diff
@@ -37,6 +37,8 @@ const (
 	// values override the default set by allAlpha.
 	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
 	allAlpha = "allAlpha"
+
+	externalTrafficLocalOnly = "externalTrafficLocalOnly"
```

Comment:
As above, should probably be initial caps letter.

Propose "ExternalBalancerLocalOnly", though this makes it sound like we are activating the feature, rather than activating the annotation to enable the feature opt-in.  @jlowdermilk ideas on names here or good enough?